hasura/metadata/databases/default/tables/public_nominees.yaml

```diff
@@ -90,71 +90,3 @@ select_permissions:
               - deleted_at:
                   _is_null: true
   role: user
-event_triggers:
```

Comment:
I think this and the cron was disabled by the seed hook: https://github.com/coordinape/coordinape/blob/a9b424a973d73f9ddc8e6ec60ffe716902dc34f7/scripts/enable_triggers.sh#L12
 but you don't actually want to commit these changes